src/common/ChatterSet.cpp

```diff
@@ -62,6 +62,22 @@ std::vector<QString> ChatterSet::filterByPrefix(const QString &prefix) const
     return result;
 }
 
+std::vector<std::pair<QString, QString>> ChatterSet::allExcept(
+    const QString &userName) const
+{
+    std::vector<std::pair<QString, QString>> result;
```

Comment:
This should be resized to `this->items.size()` to prevent reallocations, since we know an upper bound of the size.

Reply:
I'll wait on your suggestions in case we keep `allExcept`